content/concepts/transport.md

```diff
@@ -115,3 +115,88 @@ and some places in the codebase still use the "swarm" terminology.
 {{% /notice %}}
 
 [definition_switch]: /reference/glossary/#switch
+
+## QUIC
+
+QUIC is a new transport protocol that provides an always-encrypted, stream-multiplexed 
+connection built on top of UDP. It started as an experiment by Google on Google Chrome 
+in 2014, and was later standardized by the IETF in 
+[RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000).
+
+### Key challenges with TCP
+
+1. Head-of-line blocking (HoL blocking): TCP is a single byte stream exposed by the kernel, 
+   so streams layered on top of TCP experience HoL blocking.
+
+   {{% notice "info" %}}
+   HoL blocking occurs when a lost, or out-of-order delivered head packet holds up the other packets in the transmission 
+   queue. 
+   {{% /notice %}}
+
+2. Ossification: Because TCP is unencrypted, middleboxes can inspect and modify
+   TCP header fields and may break unexpectedly when they encounter anything they don’t like.
+
+3. Handshake inefficiency: the 3-way handshake is inefficient, as it spends 1-RTT on verifying 
+   the client’s address.
+
+We need a transport protocol that:
+
+- Understands streams 
+- Overcomes HOL blocking (Head-of-line blocking)
+- Overcomes the latency of connection setup
+- Overcomes the ossification risks of TCP
+
+and, ideally, keeps the guarantees of TCP.
+
```

Comment:
It would probably make sense to make this a section "Comparing HTTP/2 and HTTP/3"